editor/D3XStringEditor.cpp

```diff
@@ -165,7 +167,7 @@ void CD3XStringEditor::OnKillfocusStringedit()
 void CD3XStringEditor::OnSave() 
 {
 	ASSERT(m_modified);
-
+#if 0
```

Comment:
Can we just remove this code (and the previous one) if they're not being used?

Reply:
I prefer to just disable for now, while we don't exactly know its purpose.